chain-crypto/src/testing.rs

```diff
@@ -15,6 +15,36 @@ use rand_core::SeedableRng;
 #[derive(Clone, Debug)]
 pub struct TestCryptoGen(pub u64);
 
+pub struct TestCryptoRng(SmallRng);
```

Comment:
Please document that this is a fake crypto RNG that does not really provide cryptograhic safety.

Reply:
Done.